implementations/rust/ockam/ockam_app/src/tcp/outlet/menu_items.rs

```diff
@@ -46,7 +43,7 @@ impl TcpOutletActions {
 }
 
 /// Event listener for the "Create..." menu item
-pub fn on_create(ctx: TauriCtx) -> tauri::Result<()> {
-    let _ = create(ctx);
+pub fn on_create(app_handle: AppHandle) -> tauri::Result<()> {
+    tcp_outlet_create(app_handle);
```

Comment:
Is this/should this be a fallible operation?

Reply:
Not sure yet. Since we are only using the tray menu, right now we can't do much when something fails, other than logging the error. Perhaps we should wait until we start displaying errors to the user in popup windows.